assets/js/solcast.webflow.irr.maps.js

```diff
@@ -1,53 +1,37 @@
-// Function to fetch the video URL and update the video element
-function fetchLatestVideoUrl() {
-  // Get the div element with class mapEmbed
-  const mapEmbedDiv = document.querySelector('.mapembed');
-  if (!mapEmbedDiv) {
-    return;
-  }
+/* Global updateImageSrc */
 
-  // Extract the ID from the div
-  // const mapId = mapEmbedDiv.id;
-  // Define the API URL
-  const apiUrl = 'https://api.solcast.com.au/media/global?format=json';
+// /* Location Images */
+// Function to update the image source with the thumbnail URL from the API response
+function updateImageSrc(locationId) {
```

Comment:
Function is called in on-page script on IRR Maps pages to update thumbnails

Reply:
@Rework-Digital in that case can it go into a separate script file that is only used on that page?

The challenge we'll have going forward is identifying which scripts are used and which ones can be removed or we'll end up maintaining lots of dead code for no reason

Reply:
This script is only called on the https://solcast.io/solar-radiation-map page to set up the function, which is then run on each of the locations using an embedded script in Webflow that sets the location codes on each of the locations cards to be able to update the thumbnails to the latest one generated. 
![Screenshot 2024-06-11 120033](https://github.com/Solcast/static/assets/170994137/fab4b369-1790-4b49-ba90-34e1f7d9d8dd)

Reply:
@Rework-Digital - the <head> of the page at https://solcast.io/solar-radiation-map can't just load this script while all other pages don't include scripts they don't need?

I.e. move this into a file assets/js/solcast.solar-radiation-map.js with the scripts needed for https://solcast.io/solar-radiation-map ? Other pages shouldn't load this script since they don't need the `updateImageSrc('global')` function